src/utils/blockchainHelpers.js

```diff
@@ -116,26 +121,23 @@ const deployContractInner = (accounts, abi, deployOpts) => {
   console.log('abi', abi)
 
   const { web3 } = web3Store
-  const estimatedGasMax = 4016260
   const objAbi = JSON.parse(JSON.stringify(abi))
   const contractInstance = new web3.eth.Contract(objAbi)
+  const deploy = contractInstance.deploy(deployOpts)
 
-  return contractInstance.deploy(deployOpts).estimateGas({ gas: estimatedGasMax })
+  return deploy.estimateGas({ gas: calculateGasLimit() })
```

Comment:
Can we move `estimatedGasMax` to a constant, and use it here? So this line becomes:

```
return deploy.estimateGas({ gas: MAX_GAS_LIMIT })
```

which I feel is the intention, isn't it?

Reply:
@fvictorio done, thanks!